source/main/ipc/index.ts

```diff
@@ -8,7 +8,7 @@ import loadAsset from './load-asset';
 import getGpuStatus from './get-gpu-status';
 import { downloadManagerChannel } from './downloadManagerChannel';
 import getRecoveryWalletIdChannel from './getRecoveryWalletIdChannel';
-import { handleHardwareWalletRequests } from './getHardwareWalletChannel';
+import { handleHardwareWalletRequests } from './getHWChannel';
```

Comment:
Since you decided to rename this file, perhaps we could think about more meaningful name? Usually it should use the same name as the main export, how about `handleHardwareWalletRequests`?

Reply:
I renamed it only to prevent bloated diff. Otherwise, GH would show a lot of deleted lines (that in fact were moved to the `./hardwareWallet/index.ts`) making it hard to understand the modulik usage.

Reply:
Hmm, in this case I would keep it as is and just add a note in PR description/at top of the file producting the ugly diff. People may be searching the file by `hardwareWallet` phrase, we don't seem to abbreviate this phrase a lot.